src/blocks/location.js

```diff
@@ -75,18 +74,7 @@ export async function update(user, locality) {
     })
 
   // Update profile data.
-  const params = {
-    TableName: 'profiles',
-    Key: {
-      id: user
-    },
-    UpdateExpression: 'set locality = :locality',
-    ExpressionAttributeValues: {
-      ':locality': location
-    }
-  }
-
-  await db.update(params).promise()
-    .then(res => console.log(res))
-    .catch(e => console.log(e))
+  await profiles.updateProfile(user, 'set locality = :locality', {
```

Comment:
I'd like to change the second argument of this to an object-
```
{ locality: location }
```
and do away with the third arg.

Reply:
The reason I added those arguments is to allow there to be one updateProfile method. That particular method is being used in 4 different places with different params. So I could add some additionally logic in the method, or add specific methods for each use.